manager/integration/tests/test_settings.py

```diff
@@ -198,3 +198,29 @@ def check_tolerations_set(current_toleration_list, set_tolerations):
                  "operator": t.operator, "effect": t.effect}
 
     return current_tolerations == set_tolerations
+
+
+@pytest.skip(reason="TODO")
+def test_setting_guaranteed_engine_cpu():
+    """
+    Test setting Guaranteed Engine CPU
+
+    1. Change the setting to `0.1`
+    2. Wait for all the instance manager pods to recreated and become running
+    3. Verify every pod has guaranteed CPU set correctly to 100m
+    4. Change the setting to `0`
+    5. Wait for all the instance manager pods to recreated and become running
+    6. Verify every pod no longer has guaranteed CPU set
+    7. Change the setting to `200m`
```

Comment:
Is this an invalid value check ?

Reply:
`200m` is a valid value. It's the same as `0.2`. See https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/

Reply:
Thinking about it, yes, we should have an invalid value check.